### ISSUE TITLE
Improve stats

### ISSUE THREAD

**Issue by filinux13** [Malopieds/InnerTune#21]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

Current stats are kinda bad, it just shows the most listened songs. I wanted it to also show the time you spent listening to certain song.

### Why do you want this feature?

I want to see how much time i spent in certain song, i know this is possible because "arrange by playback time" exists, and it would be good to know that time in the settings of the song or in stats.

**Comment by Filipexos** [comment]
Also, you can add actual stats, like how much time you spent listening in last month/total...

**Comment by Malopieds** [comment]
Yep, I was thinking about doing something pretty much like [last.fm](last.fm).

**Comment by Malopieds** [comment]
Hi, I pushed some changes, can you have a look and tell me if that's what you were look for? Tell me if you think other things could be there too.

**Comment by MLZ94** [comment]
I suggest maybe for each section song artist and album, add a button ton extend the section into a full screen menu. Would be easier since i guess most people usualy have a tonnnn of artists and song and stuff😅🙏

**Comment by MLZ94** [comment]
Also, it might be a better if when we check a different chip (switching from 1 week to 1 year), it scrolls all the way to the start of the list for this periode of time. Currently, when you check the one year periode, the first item of the list remains the most listen item of the week, so there's a lonnnnng way to go to the first listen items of the year... I'm not sure i made myself clear, tell me if you need more explanation.

**Comment by fanieko** [comment]
> Hi, I pushed some changes, can you have a look and tell me if that's what you were look for?

Hi, sorry for very late reply. Yes, that was what i was looking for :)
 
> Tell me if you think other things could be there too.


I think instead of chosing each week/month/year you could also choose the interval in a calendar.

There should also be a way to search the songs to see their stats.

Also the app crashes when i click a song in stats.. 

But overall i like the design 👍

**Comment by fanieko** [comment]
I think you could also add "total time listened" for each week/month/year
